client/shipping-label/state/actions.js

```diff
@@ -140,21 +140,41 @@ const refreshPreview = ( dispatch, getState, context ) => {
 	} );
 };
 
-export const openPrintingFlow = () => ( dispatch, getState, context ) => {
+export const openPrintingFlow = () => (
+	dispatch,
+	getState,
+	context,
+	getErrors = getFormErrors
```

Comment:
I'm not a fan of changing client code just to better suit the tests. Can you instead mock `getFormErrors` in the test? Or is it too messy?